.github/workflows/ci.yml

```diff
@@ -19,3 +19,9 @@ jobs:
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
       - name: ci/test
         uses: ./.github/actions/test
+      - name: ci/test-coverage
+        if: github.event_name == 'pull_request' || github.ref_name == 'main'
```

Comment:
AFAIK, this condition is not completely complementary to `if: startsWith(github.ref_name, 'release-')`. I wonder if anything will be falling through the cracks there. 0/5.

Reply:
Yes, I don't think we need to do test:coverage on releases.  It will still do normal test on main and PR.